pkg/agent/agent.go

```diff
@@ -103,6 +97,12 @@ func (k *Klocksmith) process(stop <-chan struct{}) error {
 		return err
 	}
 
+	// we are schedulable now.
+	glog.Info("Marking node as schedulable")
+	if err := k8sutil.Unschedulable(k.nc, k.node, false); err != nil {
```

Comment:
That is an interesting question. It seems like we just indiscriminately mark nodes schedulable and not schedulable for our own uses right now, and that is something that is worth thinking about. I do think it's out of scope for this particular change though.